Packs/ProofpointIsolation/Integrations/ProofpointIsolationEventCollector/test_data/get_events_raw_response.json

```diff
@@ -0,0 +1,134 @@
+{
+    "data": [
+        {
+            "userId": "user_id_example0",
+            "userName": "user0@example.com",
+            "url": "https://exmaple.k10.com/",
+            "date": "2025-01-01T19:44:35.000+0000",
+            "region": "US",
+            "zone": "CENTRAL",
+            "classification": "CLASSIFICATION_EXAMPLE",
+            "disposition": "DISPOSITION_EXAMPLE",
+            "categories": [
+                "EXAMPLE"
+            ]
+        },
+        {
+            "userId": "user_id_example1",
+            "userName": "user1@example.com",
+            "url": "https://exmaple.k9.com/",
+            "date": "2025-01-02T19:44:35.000+0000",
+            "region": "US",
+            "zone": "CENTRAL",
+            "classification": "CLASSIFICATION_EXAMPLE",
+            "disposition": "DISPOSITION_EXAMPLE",
+            "categories": [
+                "EXAMPLE"
+            ]
+        },
+        {
+            "userId": "user_id_example2",
+            "userName": "user2@example.com",
+            "url": "https://exmaple.k8.com/",
+            "date": "2025-01-03T19:44:35.000+0000",
+            "region": "US",
+            "zone": "CENTRAL",
+            "classification": "CLASSIFICATION_EXAMPLE",
+            "disposition": "DISPOSITION_EXAMPLE",
+            "categories": [
+                "EXAMPLE"
+            ]
+        },
+        {
+            "userId": "user_id_example2",
```

Comment:
Username and userid should be the same no? 
The examples should be the same as the actual response but without secrets.

Also, I don't see examples of events that happens in the same time (which the API allows) let's test this use-case as well